.github/workflows/updatecli.yml

```diff
@@ -24,6 +24,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Checkout Epinio
```

Comment:
What does the checkout before this step do ? (See lines 24-25, here is 27).
Are both needed ?

Reply:
Yes, basically the first one is checking out the current repository where to run the CI (the `helm-charts`), while the second is checking out the `epinio` repository under the `./epinio` directory.

This is needed because now `udatecli` will check the epinio version from its tags, instead of the Github Release:

```
  epinio:
    name: "Get Latest epinio version"
    kind: "gittag"
    spec:
      path: "./epinio/"
      versionfilter:
        kind: semver
```